firstvoices/backend/resources/dictionary.py

```diff
@@ -77,15 +77,16 @@ class DictionaryEntryResource(
         ),
     )
 
-    def __init__(self, site=None):
-        if site:
-            self.site = site
+    def __init__(self, site=None, run_as_user=None):
+        self.site = site
+        self.run_as_user = run_as_user
 
     def before_import(self, dataset, **kwargs):
-        if "id" not in dataset.headers:
-            dataset.append_col(lambda x: str(uuid.uuid4()), header="id")
-        if "site" not in dataset.headers:
-            dataset.append_col(lambda x: str(self.site.id), header="site")
+        # Adding required columns, since these will not be present in the headers
```

Comment:
We can re-add the conditional here when we add the `update` feature later, but removed it for the MVP phase.